client/extensions/woocommerce/lib/keydown-callback/index.js

```diff
@@ -0,0 +1,16 @@
+// Helper function for keyboard-equivalents of onClick events
+
+/**
+ * Helper function which triggers a callback on a keydown event, only
+ * if the key pressed is space or enter - to mirror button functionality.
+ *
+ * @param {Function} callback A callback function
+ * @return {Function} the callback to fire on a keydown event
+ */
+export default function( callback ) {
```

Comment:
I agree that `onKeyDownCallback` is pretty generic for what it does, it is meant to be a keyboard accessibility handler, replacement for onClick. Happy to rename it... `generateKeyboardClickHandler` is long but basically what I was going for - thoughts?